File: benchmark/results/java-multilang/run-002/b-baseline/output/com/metaengine/demo/billing/services/InvoiceService.java
package com.metaengine.demo.billing.services;

import java.util.List;
import com.metaengine.demo.billing.aggregates.Invoice;

/** InvoiceService in the billing domain. */
public class InvoiceService {
    /** Create a new Invoice. */
    public Invoice create(Invoice input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Invoice by id. Returns null if not found. */
    public Invoice findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Invoices up to limit. */
    public List<Invoice> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Invoice by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-001/b-baseline/output/metaengine_demo/billing/services/invoice_service.py
"""InvoiceService service."""
from typing import Optional

from metaengine_demo.billing.aggregates.invoice import Invoice


class InvoiceService:
    """InvoiceService service."""

    def create(self, input: Optional[Invoice]) -> Invoice:
        raise NotImplementedError("not implemented")

    def find_by_id(self, id: str) -> Optional[Invoice]:
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[Invoice]:
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        raise NotImplementedError("not implemented")
